codegen/module_system.go

```diff
@@ -1014,6 +1062,7 @@ func (g *EndpointGenerator) ComputeSpec(
 	wg.Add(len(endpointYamls))
 	ch := make(chan endpointSpecRes, len(endpointYamls))
 	for _, yamlFile := range endpointYamls {
+		UpdateQPSLevels(yamlFile)
```

Comment:
Can we do this inside the go func(). That way you might not need a mutex below.

Reply:
I tried moving it to inside go func() but it was failing codegen tests.